### PR TITLE
Feature/add more approvers

### PR DESCRIPTION
**Pull request by IgorAlbert**
A few weeks ago there was a PR where everyone should give their feedback/approval to the new feature. Since everyone liked the feature, but could not express that he has no objection. 
It may be the right time for this PR.

Basically it works the following way, as soon as someone says "@All" in the title or description, the rebazer knows that he needs an approval from every single reviewer to merge the PR.

### CODE REVIEW

File: src/main/java/org/retest/rebazer/connector/GithubConnector.java [hunk 1]
```diff
@@ -63,11 +63,25 @@ public boolean isApproved( final PullRequest pullRequest ) {
...
 		final List<String> reviews = jsonPath.<List<String>> read( "$..state" );
 		final String titleAndDescription = pullRequest.getTitle().concat( pullRequest.getDescription() );
+		safeReviewStates( pullRequest, reviews.size(), jsonPath );
 
 		return titleAndDescription.contains( "@All" ) && !pullRequest.getReviewers().isEmpty()
 				? pullRequest.getReviewers().values().stream().allMatch( "APPROVED"::equals )
 				: pullRequest.getReviewers().values().stream().anyMatch( "APPROVED"::equals );
 	}
+
+	private static void safeReviewStates( final PullRequest pullRequest, final long reviews,
```
Comment (martin-v) [inline src/main/java/org/retest/rebazer/connector/GithubConnector.java]: Inline parameter `reviews` because it is only used in this method.

**Comment by martin-v** [comment]
closes #31

File: src/main/java/org/retest/rebazer/connector/GithubConnector.java [hunk 2]
```diff
@@ -61,7 +61,13 @@ public PullRequest getLatestUpdate( final PullRequest pullRequest ) {
 	@Override
 	public boolean isApproved( final PullRequest pullRequest ) {
 		final DocumentContext jsonPath = jsonPathForPath( requestPath( pullRequest ) + "/reviews" );
-		return jsonPath.<List<String>> read( "$..state" ).stream().anyMatch( "APPROVED"::equals );
+		final List<String> reviews = jsonPath.<List<String>> read( "$..state" );
+		final String titleAndDescription = pullRequest.getTitle().concat( pullRequest.getDescription() );
```
Comment (martin-v) [inline src/main/java/org/retest/rebazer/connector/GithubConnector.java]: create method `pullRequest.reviewByAllReviewersRequested()` to reduce code duplication.

File: src/main/java/org/retest/rebazer/connector/GithubConnector.java (hunk 1 above)
Comment (IgorAlbert) [reply src/main/java/org/retest/rebazer/connector/GithubConnector.java]: same for jsonPath.

**Comment by martin-v** [comment]
looks good

**Comment by martin-v** [comment]
@cla-bot check

**Comment by IgorAlbert** [comment]
I think we made the same changes

**Comment by martin-v** [comment]
Oh, sorry, I didn't expect that you are working :)

**Pull request merged**